### PR TITLE
Test v3

### PR DESCRIPTION
**Pull request by Sherokiel**

### CODE REVIEW

File: Calculator.php [hunk 1]
```diff
@@ -10,6 +10,12 @@
 const HISTORY = 'history';
 const AVAILABLE_COMMANDS = ['+', '-', '*', '/', '^', 'sr', QUIT, INFO, HISTORY];
 
+// ДЕНИС ПОШЛИ СМОТРЕТЬ ОБРАТНУЮ СВЯЗЬ ПОД ПИВКО!!!!!!!!!!!!!!!!!!!!!!!!
+// ДЕНИС ПОШЛИ СМОТРЕТЬ ОБРАТНУЮ СВЯЗЬ ПОД ПИВКО!!!!!!!!!!!!!!!!!!!!!!!!
+// ДЕНИС ПОШЛИ СМОТРЕТЬ ОБРАТНУЮ СВЯЗЬ ПОД ПИВКО!!!!!!!!!!!!!!!!!!!!!!!!
+// ДЕНИС ПОШЛИ СМОТРЕТЬ ОБРАТНУЮ СВЯЗЬ ПОД ПИВКО!!!!!!!!!!!!!!!!!!!!!!!!
+// ДЕНИС ПОШЛИ СМОТРЕТЬ ОБРАТНУЮ СВЯЗЬ ПОД ПИВКО!!!!!!!!!!!!!!!!!!!!!!!!
+
```
Comment (DenTray) [inline Calculator.php]: please remove it

**Review by DenTray — CHANGES_REQUESTED** [review]

**Review by netherpyro — COMMENTED** [review]
бессмысленный ПР)

**Comment by DenTray** [comment]
> бессмысленный ПР)

@netherpyro in English, please

**Comment by netherpyro** [comment]
> > бессмысленный ПР)
> 
> @netherpyro in English, please

sorry, didn't know that you are gentlemen

**Pull request closed without merging**